app/models/expression_file_info.rb

```diff
@@ -32,41 +32,47 @@ class ExpressionFileInfo
   ].freeze
   validates :modality, inclusion: { in: MODALITY_VALUES }
 
-  LIBRARY_PREPARATION_VALUES = ["10x 3' v1",
-                                "10x 3' v2",
-                                "10x 3' v3",
-                                "10x 5' v1",
-                                "10x 5' v2",
-                                "10x 5' v3",
-                                'CEL-seq2',
-                                'Drop-seq',
-                                'inDrop',
-                                'MARS-seq',
-                                'sci-RNA-seq',
-                                'Seq-Well S^3',
-                                'Seq-Well v1',
-                                'Smart-like',
-                                'Smart-seq2/Fluidigm C1',
-                                'Smart-seq2/plate-based',
-                                # non-scRNAseq Assays
-                                # single cell ATAC-seq assays
-                                'dsc-ATAC-seq',
-                                'dsci-ATAC-seq',
-                                'scATAC-seq/10x',
-                                'scATAC-seq/Fluidigm',
-                                'sci-ATAC-seq',
-                                'scTHS-seq',
-                                'snATAC-seq',
-                                # spatial transcriptomics assays
-                                '10x Visium',
-                                'MERFISH',
-                                'osmFISH',
-                                'SeqFISH+',
-                                'Slide-seq',
-                                'smFISH',
-                                'STARmap', # Note WIP for EFO term: https://broadinstitute.zendesk.com/agent/tickets/139334
-                                # single cell ChIP-seq assays
-                                'Drop-ChIP'].freeze
+  LIBRARY_PREPARATION_VALUES = [
```

Comment:
Just a note about this - making updates to any constants that are used in validation can be problematic if we remove values (which thankfully we are not doing here).  If we ever do remove/deprecate values, we will need to add migrations to update any existing documents and replace the bad values with a known good.  Otherwise, this will prevent users from updating existing matrix files due to the validation failure.  Not a concern here, but one I think we should be aware of.

Reply:
You're right, the "scATAC-seq/10x" to "10x scATAC-seq" change would fall into that category. I've made manual updates (because there were very few files with "scATAC-seq/10x". So a migration for this rare case is not necessary.